testapi.pm

```diff
@@ -430,8 +416,7 @@ is hit. Default timeout is 30s.
 
 =cut
 
-sub assert_screen {
-    my ($mustmatch) = shift;
+sub assert_screen ($mustmatch, @) {
     my $timeout;
     $timeout = shift if (@_ % 2);
```

Comment:
`@_` and `shift` is still used

Reply:
That's correct. How should I change that here specifically?

Reply:
ok. In many more places I now avoided any change but just marked the line explicitly to be not raised by the signature style check